Automattic/jetpack
# Add global notices for actions

This adds global notices to action items in the app.  

**Global Notices added for the following:** 
- Reset options 
- Jumpstart site
- Activating modules
- Deactivating modules
- Saving module setting
- Disconnecting Jetpack 
- Unlinking from WordPress.com

To Test:
- Do all of those things, make sure you see transitional and success and/or error notices.  
- Error notices should also display the error. 

![global-demo-full](https://cloud.githubusercontent.com/assets/7129409/17406562/ed2ac6ee-5a31-11e6-8437-44f5479f8605.gif)
in
